### DISCUSSION TITLE
密钥分享帖

### DISCUSSION THREAD

**Discussion by pjy612** [pjy612/SteamManifestCache#6] [Announcement]
欢迎有爱的用户分享自己手上的 depot 密钥。
确认无误后会 加入到 公共密钥库中。
由于倒卖屡禁不止，故目前仅同步给 SAI 内测用户。

调整 分享到 独立分区 [密钥清单分享](https://github.com/pjy612/SteamManifestCache/discussions/categories/%E5%AF%86%E9%92%A5%E6%B8%85%E5%8D%95%E5%88%86%E4%BA%AB)

目前 公共密钥库 数据采集自 ~[heyong5454_ManifestAutoUpdate](https://github.com/heyong5454/ManifestAutoUpdate)(删库跑路)~

额外清单密钥资源：[BlankTMing_ManifestAutoUpdate](https://github.com/BlankTMing/ManifestAutoUpdate)

已收录的游戏 也可以在此分享不存于 上面 公开的密钥库 密钥，方便其他普通用户。

成功分享 可被邀请参与内测

**Comment by pjy612** [comment]
SAI 公用密钥库 已收录 

<details>
<summary>感谢列表</summary> 
</details>

| 分享人 | 游戏名 | dlc | 
| :--- | :--- | :--- |
| 轮回 From TG  | 幽灵行者2<br>GINKA |   | 
| 苏诺丶From TG | 天际线2<br>小偷模拟器2 |  |
| 不留名的好心人 From TG | ARK: Survival Ascended | |
| @ZEERDEER From TG | [WRC Generations - The FIA WRC Official Game](https://github.com/pjy612/SteamManifestCache/discussions/6#discussioncomment-7414978) |
| 部 From QQ 厄小幺游戏分享群  | 很多 | 很多



感谢 上述分享者 造福大众 。

维护列表编辑有点麻烦 后面直接按跟帖来吧。。。
**不过 所有分享的游戏 只能保证 密钥有收录，并不能保证 后续持续更新**

**Comment by ZEERDEER** [comment]
**WRC Generations – The FIA WRC Official Game**
```
appid 1953520
depotid 1953521
manifest & DecryptionKey
```

[WRC Generations - The FIA WRC Official Game.zip](https://github.com/pjy612/SteamManifestCache/files/13197729/WRC.Generations.-.The.FIA.WRC.Official.Game.zip)

**Reply by pjy612** [reply]
感谢 验证成功 已收录  
![image](https://github.com/pjy612/SteamManifestCache/assets/4072526/ad8887b4-8bb8-4e65-932c-cb75871479a1)

**Reply by pjy612** [reply]
已公开

**Comment by ZEERDEER** [comment]
# Forza Motorsport (极限竞速)

```
appid 2440510
depotid 2440511
```
manifest & DecryptionKey
[Forza Motorsport.zip](https://github.com/pjy612/SteamManifestCache/files/13241889/Forza.Motorsport.zip)

**Reply by pjy612** [reply]
感谢分享

**Reply by pjy612** [reply]
已公开

**Comment by pjy612** [comment]
# 1324340 Made in Abyss: Binary Star Falling into Darkness （来自深渊）
[来自深渊.zip](https://github.com/pjy612/SteamManifestCache/files/13259607/default.zip)
from TG Eric

**Reply by pjy612** [reply]
已公开

**Comment by pjy612** [comment]
# 930910  STEINS;GATE: Linear Bounded Phenogram
# 970560  STEINS;GATE: My Darling's Embrace
[石头门.txt](https://github.com/pjy612/SteamManifestCache/files/13262339/default.txt)

from TG Eric

**Reply by pjy612** [reply]
已公开

**Comment by pjy612** [comment]
# 2238900 星之海洋2
[星之海洋2 第二个故事 R 清单+key.zip](https://github.com/pjy612/SteamManifestCache/files/13263525/2.R.%2Bkey.zip)
from TG cuxk

**Reply by pjy612** [reply]
已公开

**Reply by fuyutsuki11** [reply]
让我来康康

**Comment by LGCaerwyn** [comment]
Back to the dawn 动物谜城
`appid 1735700
depotid 1735701
manifest & DecryptionKey`
[back to the dawn.zip](https://github.com/pjy612/SteamManifestCache/files/13267204/back.to.the.dawn.zip)

**Reply by pjy612** [reply]
已公开

**Reply by Scropiouos** [reply]
公共库中未收录密钥？

**Comment by ZEERDEER** [comment]
COCOON (茧)

《COCOON》由《LIMBO》和《INSIDE》的首席遊戲設計師 Jeppe Carlsen 傾力打造，
將帶你踏上冒險之旅，在世界之間跳躍穿梭。
掌握穿越世界的技巧，破解錯綜複雜的謎題，揭開宇宙奧妙之謎。

```
appid 1497440
depotid 1497441
```
manifest & DecryptionKey
[COCOON.zip](https://github.com/pjy612/SteamManifestCache/files/13298498/COCOON.zip)

**Reply by pjy612** [reply]
已收录并公开

**Reply by Scropiouos** [reply]
好像DecryptionKey没有正常写入，仍然显示无法获取

**Comment by ZEERDEER** [comment]
### Zodiacats

跟隨一只名叫凱瑟琳的貓踏上尋找遺失的占星莎草紙的旅途。了解她作為女巫助手和給地球帶來正能量的職責！讓凱瑟琳引導你使用占星盤，向你展示每個星座的特質。你也可以探知你的小貓的星座！


appid ```1817000```
depotid ```1817001```

[Zodiacats.zip](https://github.com/pjy612/SteamManifestCache/files/13306952/Zodiacats.zip)

**Reply by pjy612** [reply]
已收录并公开

**Comment by LGCaerwyn** [comment]
Hero's Adventure: Road to Passion
大侠立志传：碧血丹心
`appid 1948980`
`depotid 1948981`
[大侠立志传碧血丹心.zip](https://github.com/pjy612/SteamManifestCache/files/13390078/default.zip)

**Reply by pjy612** [reply]
感谢分享，不过这个在两个仓库应该都有了。

**Reply by pjy612** [reply]
留下 内测码 和 邮箱 或 tg用户名，推荐 github 设置个 对外显示的邮箱
PS.竟然还有神杀大佬？
@LGCaerwyn

**Comment by Scropiouos** [comment]
Darkest Dungeon® II: The Binding Blade
`appid 1940340`
`depotid 2598510`
[Darkest Dungeon II——The Binding Blade.zip](https://github.com/pjy612/SteamManifestCache/files/13665655/Darkest.Dungeon.II.The.Binding.Blade.zip)

**Reply by Scropiouos** [reply]
能不能有个内测资格

**Reply by pjy612** [reply]
留下 内测码 和 邮箱 或 tg用户名，推荐 github 设置个 对外显示的邮箱
@Scropiouos

**Reply by pjy612** [reply]
emmm 需要
2598510_6047719762198650119

**Comment by yehuailin** [comment]
Dredgers-沉没的王国
appid 1112930
depotid 228990
[Dredgers.zip](https://github.com/pjy612/SteamManifestCache/files/13666918/Dredgers.zip)

**Reply by yehuailin** [reply]
看了下库里应该是没有的，能不能有个内测资格

**Reply by pjy612** [reply]
留下 内测码 和 邮箱 或 tg用户名，推荐 github 设置个 对外显示的邮箱
@yehuailin

**Reply by yehuailin** [reply]
1572503744@qq.com
tg 同 git

**Reply by pjy612** [reply]
新版 SAI 运行的时候 会拿到 内测码，记得查收TG消息。
群组的话 tools 官网和论坛里 应该有写唯一群组。

**Reply by pjy612** [reply]
请查收

**Comment by llllllu91** [comment]
Tooth and Tail 尾牙
appid 286000
depotid 286001
[Tooth and Tail.txt](https://github.com/pjy612/SteamManifestCache/files/13667543/Tooth.and.Tail.txt)

我看有清单、但是没key，不知道可以不可以

**Reply by pjy612** [reply]
留下 内测码 和 邮箱 或 tg用户名，推荐 github 设置个 对外显示的邮箱
@llllllu91

**Reply by llllllu91** [reply]
邮箱公开了

**Reply by llllllu91** [reply]
tg lllllllu91

**Comment by llllllu91** [comment]
Warriors of the Nile 尼罗河勇士
appid 1267470
depotid 1267471
[Warriors of the Nile.txt](https://github.com/pjy612/SteamManifestCache/files/13667692/Warriors.of.the.Nile.txt)

**Comment by llllllu91** [comment]
Ash of Gods Redemption  诸神灰烬 抉择
appid 691690
depotid 691691
              820860
[Ash of Gods Redemption.txt](https://github.com/pjy612/SteamManifestCache/files/13667767/Ash.of.Gods.Redemption.txt)

**Comment by llllllu91** [comment]
The Legend of Heroes 英雄传说6英文版 三部曲
appid
FC 251150 
SC 251290
3RD 436670
depotid
FC 251151
SC 251291
3RD 436671
[The Legend of Heroes.txt](https://github.com/pjy612/SteamManifestCache/files/13667833/The.Legend.of.Heroes.txt)

**Comment by zerodeep** [comment]
Aaero
appid 499890
depotid 499891
[Aaero.txt](https://github.com/pjy612/SteamManifestCache/files/13670804/Aaero.txt)
[Aaero清单.zip](https://github.com/pjy612/SteamManifestCache/files/13671037/Aaero.zip)

Warno
appid 1611600
depotid 1611601
[Warno.zip](https://github.com/pjy612/SteamManifestCache/files/13670993/Warno.zip)

**Reply by pjy612** [reply]
由于没有收录，所以 最好可以编辑下。
带上一组配套清单
记得留下 内测码

**Reply by zerodeep** [reply]
我不知道清单是哪个一股脑全拉进去了

**Reply by zerodeep** [reply]
请问有什么方法可以批量导出清单和秘钥吗

**Reply by pjy612** [reply]
> 请问有什么方法可以批量导出清单和秘钥吗

看看隔壁 wxy大佬的项目了

**Reply by pjy612** [reply]
注意查收邮件，如果没收到或者有问题，可以加入 tools tg群 用 tg沟通。
能用的话 记得回复一下

**Comment by karson0014** [comment]
Move or Die      妄想症：Deliver Me     a way out   太吾绘卷 The Scroll Of Taiwu   Mr. Shifty

第一次操作 不知道对不对。 望指正。

[five.zip](https://github.com/pjy612/SteamManifestCache/files/13671334/five.zip)

**Reply by pjy612** [reply]
貌似不对 不过你应该 可以重新 编辑 自己的内容

**Reply by karson0014** [reply]
文件应该可以下载了 不知道内容是否有问题

**Reply by pjy612** [reply]
问题不大，这些都有收录，但是密钥应该能方便他人。

**Comment by karson0014** [comment]
只有这一个可能小众点的了 别的我估计早就收录了 可否给个资格/(ㄒoㄒ)/~~
[Remember Me.zip](https://github.com/pjy612/SteamManifestCache/files/13671790/Remember.Me.zip)

**Reply by pjy612** [reply]
你不发内测码 我也没办法授权啊

**Reply by karson0014** [reply]
BB53DAF5525642624B3301C2A7DEC557E0CE6392 

麻烦了

**Reply by pjy612** [reply]
注意查收邮件

**Reply by karson0014** [reply]
已收到 非常感谢

**Comment by sika15** [comment]
[Cyberpunk 2077.zip](https://github.com/pjy612/SteamManifestCache/files/13673836/Cyberpunk.2077.zip)
[Uploading 往日之影DLC.zip…]()
Cyberpunk 2077
内测码：92D131C97EEA3EB4C8C27AEF89C0FABCBCD498BC

**Reply by sika15** [reply]
我没有找到你需要的文件

**Reply by pjy612** [reply]
貌似 可以重新传一下

**Reply by sika15** [reply]
我找找看你需要的文件

**Reply by sika15** [reply]
试试看这个？

**Reply by pjy612** [reply]
注意查收邮件

**Comment by ArtMoe** [comment]
The DioField Chronicle 神领编年史
appid 1399080
depotid 1399081
内测码 0B31D8099025E9EB95C9354FC7FDEAA00654B094
[The DioField Chronicle.zip](https://github.com/pjy612/SteamManifestCache/files/13673508/The.DioField.Chronicle.zip)
第一次分享，不知道可不可以

**Reply by pjy612** [reply]
注意查收邮件

**Comment by AKAmars** [comment]
内测码：F0AD50283F85A55B6EE796812A63693BEAA22C50
樱姬
appid:1732740
下载秘钥：
```
1732741;882a2f7e5e0604fd2848ed8459f6889fb4d91a19288b10abb196b1e4db6dbf2b
1777420;99700144749cb71563e59d1365a08f98b71a361fca07bd3d30c266773e1fa325
```
appid:1568940
```
1568941:61cc8ee934cfe2ac41166e0d7ca0c45da5d9af38ac852a498f13bb0e054a270d
```
DLC
appid:1715900
```
1715900:7bc678113da51d6586e69b871b57860cbabd21d99bd099f662d5b8d71893e128
```

**Reply by pjy612** [reply]
如何联系？ 推荐 弄个同名的 tg 或者 github 上 公开下邮箱

**Comment by muxikuku** [comment]
**_Furi_**
**appid** 

> 423230

**depotid** 

> 423231

`内测码:58A02DE597C70BEF5C62567B9391A3ECDAB19717`
[Furi.zip](https://github.com/pjy612/SteamManifestCache/files/13674575/Furi.zip)
分享一个自己一直在玩的硬核格斗游戏，同时希望爱分享的朋友们越来越多~^ ^

**Reply by pjy612** [reply]
注意查收邮件

**Comment by chunge35** [comment]
严阵以待（ready or not）今天刚更新的正式版，申请加入内测，内测码：5B9E46D8F55061729179A11F9994B349260005AD
[ready or not.zip](https://github.com/pjy612/SteamManifestCache/files/13674721/ready.or.not.zip)

**Reply by pjy612** [reply]
如何联系？ 推荐 弄个同名的 tg 或者 github 上 公开下邮箱
